### PR TITLE
chore: polishing repo overall look

### PR DESCRIPTION
**Pull request by sachaarbonel**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,288 @@
...
+
+// Remove an activity by its id
+await user1.removeActivityById('e561de8f-00f1-11e4-b400-0cc47a024be0');
+// or remove by the foreign id
+await user1.removeActivityByForeignId('tweet:1');
+
+// mark a notification feed as read
+  await notification1.getActivities(
+    marker: ActivityMarker().allRead(),
+  );
```
Comment (xsahil03x) [inline README.md:123]: indentation

File: README.md [hunk 2]
```diff
@@ -1,7 +1,288 @@
...
+
+// Stop following another feed
+await user1.unfollow(client.flatFeed('flat', '42'));
+
+// Stop following another feed while keeping previously published activities
+// from that feed
+await user1.unfollow(client.flatFeed('flat', '42'), keepHistory: true);
+
+  // Follow another feed without copying the history
+  await user1.follow(client.flatFeed('flat', '42'), activityCopyLimit: 0);
```
Comment (xsahil03x) [inline README.md:142]: indentation

File: README.md [hunk 3]
```diff
@@ -1,7 +1,288 @@
...
+
+// adding multiple activities
+  const activities = [
+    Activity(actor: '1', verb: 'tweet', object: '1'),
+    Activity(actor: '2', verb: 'tweet', object: '3'),
+  ];
+  await user1.addActivities(activities);
+
+ // specifying additional feeds to push the activity to using the to param
+// especially useful for notification style feeds
```
Comment (xsahil03x) [inline README.md:159]: indentation

File: packages/stream_feed/example/main2.dart [hunk 4]
```diff
@@ -0,0 +1,99 @@
+import 'package:stream_feed/stream_feed.dart';
```
Comment (xsahil03x) [inline packages/stream_feed/example/main2.dart:1]: Is this file the replacement of the original main.dart? If so can you rename this.

File: packages/stream_feed/lib/src/core/models/follow.dart [hunk 5]
```diff
@@ -7,20 +7,21 @@ part 'follow.g.dart';
 @JsonSerializable()
 class Follow extends Equatable {
   ///
-  const Follow(this.feedId, this.targetId);
+  const Follow(
```
Comment (xsahil03x) [inline packages/stream_feed/lib/src/core/models/follow.dart:10]: remove todo

File: packages/stream_feed/example/main2.dart (hunk 4 above)
Comment (sachaarbonel) [reply packages/stream_feed/example/main2.dart:1]: yeah I used it to do a small round of QA for the readme snippets not sure if we should remove it and fix main.dart, or keep it and name it snippets.dart for example

Comment (sachaarbonel) [reply packages/stream_feed/example/main2.dart:1]: I deleted it let's fix main.dart in another PR

**Pull request merged**